engine/src/main/java/io/camunda/zeebe/engine/metrics/ProcessEngineMetrics.java

```diff
@@ -20,6 +20,9 @@
   private static final String ACTION_COMPLETED = "completed";
   private static final String ACTION_TERMINATED = "terminated";
 
+  private static final String ACTION_EVALUATED_SUCCESSFULLY = "evaluated_successfully";
+  private static final String ACTION_EVALUATED_FAILED = "evaluated_failed";
```

Comment:
🔧 `evaluated_failed` is consistent with `evaluated_successfully`, but it's a bit strange to read. I don't see an alternative that is better. For example, I thought of `evaluation_succeeded` and `evaluation_failed` which are consistent, but succeeded sounds strange. If you do happen to find better alternatives then please adjust it.